### PR TITLE
Add comments and remove any unused variables

### PR DESCRIPTION
**Pull request by penandlim**
* Add natspec comments
* Update openzepplin imports to 4.7.3
* Remove unused variables
* Change revert to require for more context on debugging.

### CODE REVIEW

**Review by mjdatadog — APPROVED** [review]

File: contracts/xchainGauges/AnyCallTranslator.sol [hunk 1]
```diff
@@ -88,17 +103,28 @@ contract AnyCallTranslator is OwnableUpgradeable {
...
+    /// @param _anyCallContract address of the AnyCallV6 contract
     function setAnyCall(address _anyCallContract) external onlyOwner {
         anyCallContract = _anyCallContract;
         anyCallExecutor = ICallProxy(_anyCallContract).executor();
     }
 
+    /// @notice withdraw any ETH that was sent to AnyCall contract
+    /// @dev Only callable by owner
+    /// @param _amount amount of ETH to withdraw
     function withdraw(uint256 _amount) external onlyOwner {
```
Comment (hannespfeiffer) [inline contracts/xchainGauges/AnyCallTranslator.sol:117]: How would we get the ETH out of this contract, once we've gotten it out of the AnyCall contract?

File: contracts/xchainGauges/AnyCallTranslator.sol [hunk 2]
```diff
@@ -141,6 +176,11 @@ contract AnyCallTranslator is OwnableUpgradeable {
...
     }
 
+    /// @notice Receive a cross chain call via anyCallV6 and execute a call on this network
+    /// @dev Only callable anyCallExecutor, the executor contract of anyCallV6
+    /// @param toAndData abi encoded target address and function calldata to execute
+    /// @return boolean indicating success of the call
+    /// @return bytes any data returned from the call
     function anyExecute(bytes calldata toAndData)
         external
         returns (bool, bytes memory)
```
Comment (hannespfeiffer) [inline contracts/xchainGauges/AnyCallTranslator.sol:186]: Line 195: should change comment to something like "Check that caller is AnyCallTranslator on source chain", maybe?

File: contracts/xchainGauges/bridgers/ArbitrumBridger.sol [hunk 3]
```diff
@@ -128,22 +150,34 @@ contract ArbitrumBridger is Bridger {
         }
     }
 
+    /// @notice Check if this bridged can be used=
```
Comment (hannespfeiffer) [inline contracts/xchainGauges/bridgers/ArbitrumBridger.sol:153]: *bridger *=

File: contracts/xchainGauges/bridgers/OptimismBridger.sol [hunk 4]
```diff
@@ -76,19 +92,31 @@ contract OptimismBridger is Bridger {
...
 
+    /// @notice Set gas limit to use for bridging
+    /// @dev The function can only be called by the contract owner
+    /// @param _gasLimit New gas limit
     function setGasLimit(uint32 _gasLimit) external onlyOwner {
         emit UpdateGasLimit(gasLimit, _gasLimit);
         gasLimit = _gasLimit;
     }
 
+    /// @notice Set L2 token address for the given L1 token
```
Comment (hannespfeiffer) [inline contracts/xchainGauges/bridgers/OptimismBridger.sol:116]: maybe adding "and approve bridge" under @dev?

**Review by hannespfeiffer — COMMENTED** [review]
Very thorough and clear comments, only q I have is how we get ETH out of the `AnyCallTranslator` contract, in case we `withdraw()` from the `AnyCall` contract

File: contracts/xchainGauges/AnyCallTranslator.sol (hunk 1 above)
Comment (penandlim) [reply contracts/xchainGauges/AnyCallTranslator.sol:117]: `rescue` function can be used for ETH withdraw from this contract.

**Pull request merged**